src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -156,6 +165,19 @@ export default class ReplicaSetDetailController {
    */
 
   /**
+   * Updates replicas count in replica set
+   * @param {number} replicasCount
+   * @param {function(!backendApi.ReplicaSetDetail)=} opt_callback
+   * @param {function(!angular.$http.Response)=} opt_errback
+   * @private
+   */
+  updateReplicas_(replicasCount, opt_callback, opt_errback) {
+    this.replicaSetDetail.podsDesired = replicasCount;
+    this.replicaSetDetailResource_.save(this.replicaSetDetail, opt_callback, opt_errback);
+    // TODO(floreks): Think about refreshing data on this page after update.
```

Comment:
I don't plan to add this on this PR. I was just investigating possible solutions. Refreshing after fixed time does not work very well when we are scaling down number of pods. Cleaning up and terminating pods takes sometimes over 10s even on my local cluster. Scaling up on the other hand is fast.